packages/adapters/src/adapters/Cornerstone3D/CircleROI.ts

```diff
@@ -183,7 +183,7 @@ class CircleROI extends BaseAdapter3D {
         points.push({ x: end[0], y: end[1], z: center[2] });
 
         const cachedStatsKeys = Object.keys(cachedStats)[0];
-        const { area, radius } = cachedStatsKeys
+        const { area = undefined, radius = 0 } = cachedStatsKeys
```

Comment:
You don't need to assign undefined, it is implicit

Reply:
It doesn't build without for some reason, docs have been broken for months now.